docs/en/developers/03_Backend/05_Extensions.md

```diff
@@ -178,7 +178,10 @@ The following events are available:
 * `nav_menu` (`function() -> string`): will be executed if the navigation was built.
 * `nav_reading_modes` (`function($reading_modes) -> array | null`): **TODO** add documentation.
 * `post_update` (`function(none) -> none`): **TODO** add documentation.
-* `simplepie_before_init` (`function($simplePie, $feed) -> none`): **TODO** add documentation.
+* `simplepie_after_init` (`function(\SimplePie\SimplePie $simplePie, FreshRSS_Feed $feed, bool $result) -> none`): Triggered after fetching an RSS/Atom feed with SimplePie. Useful for instance to get the HTTP response headers (e.g. `$simplePie->get_headers();`).
```

Comment:
The function `get_headers()` would be missing.  
Currently they'd need to be fetched with `$simplePie->data['headers']`

Here's the assignment and in the rest of the class they're accessed the same way  
https://github.com/FreshRSS/FreshRSS/blob/966f211202bc2ed6bf56b64e1ea3c4804e93c404/lib/simplepie/simplepie/src/SimplePie.php#L1877

Reply:
Thanks; doc fixed